x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/privileges/11_builtin.yml

```diff
@@ -16,4 +16,4 @@ setup:
   # I would much prefer we could just check that specific entries are in the array, but we don't have
   # an assertion for that
   - length: { "cluster" : 62 }
-  - length: { "index" : 22 }
+  - length: { "index" : 24 }
```

Comment:
This test has been [disabled in release builds](https://github.com/elastic/elasticsearch/blob/939dc8bb8ed3b2e13919fc96383377985aa4ccf1/x-pack/plugin/build.gradle#L54) for some time. I'll depend on that for now. We can remove it from the blacklist after feature flag removal.